src/components/notebook-utils.js

```diff
@@ -14,15 +14,7 @@ import * as Utils from 'src/libs/utils'
 import validate from 'validate.js'
 
 
-export const notebookLockHash = async (bucketName, email) => {
-  const msgUint8 = new TextEncoder().encode(`${bucketName}:${email}`)
-  const hashBuffer = await crypto.subtle.digest('SHA-256', msgUint8)
-  const hashArray = new Uint8Array(hashBuffer)
-  return _.flow(
-    _.map(v => v.toString(16).padStart(2, '0')),
-    _.join('')
-  )(hashArray)
-}
+export const notebookLockHash = (bucketName, email) => Utils.sha256(`${bucketName}:${email}`)
```

Comment:
This seems much cleaner, but I noticed we are now seeing notebooks test failures. Usually I'd assume a flake, but since we've made a change here I would want to double check to make sure its not a real issue. I could see an async issue. It looks functionally equivalent though, so I would be surprised.